pkg/libvirt/resolve_vm.go

```diff
@@ -43,6 +42,7 @@ var (
 )
 
 func GetVMID(pid uint64) (string, error) {
+	procPath := config.ProcDir() + "/%d/cgroup"
 	fileName := fmt.Sprintf(procPath, pid)
```

Comment:
combine the two statements in to same sprintf ?

Reply:
💯  ... I will clean this up in a followup patch which addresses a bigger issue.

Reply:
Done